src/services/peerconnection.ts

```diff
@@ -150,7 +151,12 @@ export class PeerConnectionHelper {
                             // situations which certainly would lead to ugly race conditions.
                             this.connectionFailedTimer = null;
                             this.log.debug('ICE connection considered failed');
-                            this.pc.close();
+
+                            // Close and raise disconnected event
+                            // Note: We need to simulate a 'closed' event here due to browser inconsistencies
+                            this.closed = true;
```

Comment:
The PeerConnectionHelper is never reused, right? (Asking since `closed` is never reset.)

Reply:
Yup, never reused.